src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -59,8 +63,16 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
     public static List<Index> parseIndexes(String oneBasedIndexes) throws ParseException {
         String[] trimmedIndexes = oneBasedIndexes.trim().split(" ");
         List<Index> indexList = new ArrayList<>();
+        Set<String> uniqueStringSet = new HashSet<>();
+
         for (String index : trimmedIndexes) {
-            indexList.add(parseIndex(index));
+            if (uniqueStringSet.contains(index)) {
+                logger.warning(String.format(MESSAGE_DUPLICATE_INDEX, index));
```

Comment:
Great job adding a check for duplicate indices to this method!

Reply:
Thanks!